tools/trap/HowToUseTrap.md

```diff
@@ -44,6 +44,303 @@ Hardware Configuration > Board Selection -> Reset a board on assert status autom
+0x21813c4	 0xc51   	 app2    	  main	/root/tizenrt/loadable_apps/loadable_sample/wifiapp/wifiapp.c:57
+0x21813d0	 0xe001c2c	 kernel  	  task_start	/root/tizenrt/os/kernel/task/task_start.c:133
+
+4. Miscellaneous information:
+-----------------------------------------------------------------------------------------
+```
+ex)
+$ python3 ramdumpParser.py -t ./logs
+
+Example Call Stack Output for Kernel crash is as follows:
+```
+*************************************************************
+dump_file                   : None
+log_file                    : kernelassert
+Number of binary            : 1 [kernel]
+"kernel" elf_file           : ../../build/output/bin/tinyara.axf
+*************************************************************
+
+-----------------------------------------------------------------------------------------
+1. Crash Binary             : kernel
+
+2. Crash point (PC or LR)
+
+	[ Caller - return address (LR) - of the function which has caused the crash ]
+	- symbol addr        : 0x0e001c1b
+	- function name      : setbasepri
+	- file               : /root/tizenrt/os/include/arch/armv8-m/irq.h:247
+
+	[ Current location (PC) of assert ]
+	- symbol addr        : 0x10010f5e
+	- function name      : os_start
+	- file               : /root/tizenrt/os/kernel/init/os_start.c:609
+
+	[ Exact crash point might be -4 or -8 bytes from the PC ]
+	- symbol addr of (pc - 4)       : 0x10010f5a
+	- function name of (pc - 4)     : os_start
+	- file of (pc - 4)              : /root/tizenrt/os/kernel/init/os_start.c:609
+	- symbol addr of (pc - 8)       : 0x10010f56
+	- function name of (pc - 8)     : os_start
+	- file of (pc - 8)              : /root/tizenrt/os/kernel/init/os_start.c:607 (discriminator 1)
+
+3. Call stack of last run thread using Stack dump
+
+	- Current stack pointer:	 0x10004f20
+
+Stack_address	 Symbol_address	 Symbol location  Symbol_name		File_name
+0x10004f30	 0xe0060b8	 kernel  	  lowvsyslog	/root/tizenrt/os/include/syslog.h:263
+0x10004f38	 0xe0067bc	 kernel  	  __ultoa_invert	/root/tizenrt/lib/libc/stdio/lib_ultoa_invert.c:64
+0x10004f3c	 0xe005eb4	 kernel  	  lib_noflush	/root/tizenrt/os/include/tinyara/streams.h:605
+0x10004f68	 0xe008750	 kernel  	  up_usagefault	/root/tizenrt/os/arch/arm/src/armv8-m/up_usagefault.c:84
+0x10004f78	 0x100123b4	 kernel  	  up_doirq	/root/tizenrt/os/arch/arm/src/armv8-m/up_doirq.c:91
+0x10004f90	 0x10012094	 kernel  	  exception_common	/root/tizenrt/os/arch/arm/src/armv8-m/up_exception.S:156
+0x10004fd4	 0x10012670	 kernel  	  up_unblock_task	/root/tizenrt/os/include/tinyara/arch.h:472
+0x10004fd8	 0xe001bcc	 kernel  	  task_activate	/root/tizenrt/os/include/sched.h:141
+0x10004fdc	 0x10010db8	 kernel  	  os_start	/root/tizenrt/os/include/tinyara/init.h:89
+
+4. Miscellaneous information:
+-----------------------------------------------------------------------------------------
+```
+ex)
+python3 ramdumpParser.py -t ./logfile
+
+
```

Comment:
Missing belows.

Example Call Stack Output for Common Binary crash is as follows:
\`\`\`

Reply:
Could you check md file on github? Current version looks strange because of above.

Reply:
Updated